src/theme.ts

```diff
@@ -8,6 +8,7 @@ import baseStyled, { ThemedStyledInterface, createGlobalStyle, useTheme } from '
 // const LeagueSpartanBold = require('./fonts/LeagueSpartan-Bold.otf');
 
 const DarkTheme = {
+  OVERLAY: 'rgba(17,17,17,0.50)',
```

Comment:
added it like this as it didn't fit anywhere imo and i doubt we will have more than one overlay variation